main.go

```diff
@@ -92,8 +97,14 @@ func main() {
 		"Show version information")
 	flag.StringVar(&printImages, "print-images", "",
 		"Print the default images the operator could deploy and exit. Possible values: list")
+	flag.StringVar(&printCalicoCRDs, "print-calico-crds", "",
```

Comment:
Could you just drop requiring "all" if you want to print all, and just call this function with no parameters? I think doing that should result in the same effect as calling all since it will check if the file has a prefix of nothing, which is always true.